.github/workflows/build-and-publish-master.yaml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
+      - arh-fix-publish-master
```

Comment:
I added this to test the action before merging, will remove once I see it works.